src/main/java/appeng/me/cache/CraftingGridCache.java

```diff
@@ -484,7 +484,8 @@ public StorageChannel getChannel()
 				{
 					if( ais.getItem() == whatToCraft.getItem() && ( !ais.getItem().getHasSubtypes() || ais.getItemDamage() == whatToCraft.getItemDamage() ) )
 					{
-						if( details.isValidItemForSlot( slotIndex, ais.getItemStack(), world ) )
+						// TODO: check if OK
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)